docs/appkit/javascript/ethers/about/implementation.mdx

```diff
@@ -11,20 +11,17 @@ import { mainnet, arbitrum } from '@reown/appkit/networks'
 // 1. Get projectId from https://cloud.reown.com
 const projectId = 'YOUR_PROJECT_ID'
 
-// 2. Set up Ethers adapter
-const ethersAdapter = new EthersAdapter()
-
-// 3. Create your application's metadata object
+// 2. Create your application's metadata object
 const metadata = {
   name: 'AppKit',
   description: 'AppKit Example',
   url: 'https://reown.com/appkit', // origin must match your domain & subdomain
   icons: ['https://avatars.githubusercontent.com/u/179229932']
 }
 
-// 4. Create a AppKit instance
+// 3. Create a AppKit instance
 const modal = createAppKit({
-  adapters: [ethersAdapter],
+  adapters: [new EthersAdapter()],
```

Comment:
Is there a specific issue with the code snippets on the docs earlier? I think that should work the same way. Can you confirm?

Reply:
IMO mainly less lines and less steps always is better for a first view on how to implement it
https://reown-docs-git-fix-several-fix-over-docs-reown-com.vercel.app/appkit/react/core/installation#implementation